### ISSUE TITLE
Black doesn't split the line with string and format()

### ISSUE THREAD

**Issue by jaklan** [python/black#581]
Operating system: Ubuntu 16.04
Python version: 3.6.6
*Black* version: 18.9b0
Does also happen on master: yes

Length limit: 80

Before Black:
```python
logging.info("Evaluating the model with {} of the supplied data held out for tests ({}).".format(sup_opts["test_split"], len(test)))
```

After Black:
```python
logging.info(
    "Evaluating the model with {} of the supplied data held out for tests ({}).".format(
        sup_opts["test_split"], len(test)
    )
)
```
The line with string should be splitted, because it exceeds the line length limit, but it's not. The same effect is for different settings, e.g. 60 chars, 50 chars.

**Comment by JelleZijlstra** [comment]
This seems the same as #182.

**Comment by wbolster** [comment]
this should be closed as a dup of #182  indeed
